src/dom/index.js

```diff
@@ -198,3 +201,24 @@ export const scroll = (node) => {
 
   return { get, to, lock, unlock };
 };
+
+
+/**
+ * @function isMobileTouchDevice
+ * @description Checks if user client is a mobile device
+ * @returns {boolean}
+ */
+export const isMobileTouchDevice = () => {
```

Comment:
Check the rest of the file. As far as I can see all those functions don't use any globals - they might not be available. Better to pass them in. Sorry for not noticing earlier. Just pass in `window` and `navigator`? Or maybe just one of them if you can get the other through it